src/components/Details.js

```diff
@@ -0,0 +1,161 @@
+                item.components.o3, item.components.so2, item.components.nh3,
+                item.components.pm2_5, item.components.pm10],
+              backgroundColor: ['red', 'blue', 'green', 'orange',
+                'yellow', 'purple', 'pink', 'brown'],
+            },
+          ],
+        }
+        }
+              options={{
+                plugins: {
+                  legend: {
+                    display: true,
+                    position: 'bottom',
+                    labels: {
+                      color: '#fff',
+                      fontSize: 18,
+                      fontWeight: 700,
+                    },
+                  },
+                  title: {
+                    display: true,
+                    text: 'Air pollution Data',
+                    color: '#fff',
+                  },
+                },
+              }}
+            />
+            <p className="aqi-detail">
+              AQI(Air Quality Index):&nbsp;
+              {rating(item.main.aqi)}
+            </p>
+          </div>
+        ))
+      );
+    }
+    return (
+      <div className="pollution-data">
+        <Pie
+          type="pie"
+          data={
+      {
+        labels: ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'],
+        datasets: [
+          {
+            data: [12, 12, 12, 12, 12, 12, 12, 12],
+            backgroundColor: ['red', 'blue', 'green', 'orange',
+              'yellow', 'purple', 'pink', 'brown'],
+          },
+        ],
+      }
+      }
+          options={{
+            plugins: {
+              legend: {
+                display: true,
+                position: 'bottom',
+                labels: {
+                  color: '#fff',
+                  fontSize: 18,
+                },
+              },
+              title: {
+                display: true,
+                text: 'loading data from the server...',
+                color: '#fff',
+              },
+            },
+          }}
+        />
+      </div>
+    );
+  };
+
+  return (
+    <div className="container">
+      <h2>{countryname}</h2>
+      <section className="details-section">
+        {countries.map((country) => {
+          if (country.name.common === countryname) {
+            const countryCode = country.cca2.toLowerCase();
+            return (
+              <img
+                key={countryCode}
+                src={`${countryMap}${countryCode}/vector.svg`}
+                className="country-map"
+                alt="country map"
+              />
+            );
+          }
+          return null;
+        })}
+        <div className="data">
+          {weather.map((item) => (
+            <figure key={item.id}>
+              <img src={baseImageUrl + item.icon + imageExtension} alt="weather icon" />
+              <figcaption>{item.description}</figcaption>
+            </figure>
+          ))}
+          {renderPollutionData(airPollution)}
+        </div>
+      </section>
+    </div>
```

Comment:
- [ ] Great job @PinkMoon25 ! :clap: . So that you may match the provided style guidelines, I suggest you make the gas emissions in a tabular form. This way, you will have met all the style guidelines and definitely met all the project requirements for the details page.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/188139261-0f41d56d-7cf3-481a-b0b3-5b80ab313770.png)


**This is my suggested design as per the provided design guidelines:**

![image](https://user-images.githubusercontent.com/86318284/188139361-8b8dfe37-0600-4f22-87b6-66e4f8754a75.png)

Reply:
Thanks @Nemwel-Boniface I have made the necessary changes.